xedocs/schemas/corrections/bayes_model.py

```diff
@@ -0,0 +1,9 @@
+"""
+
+"""
+
+from .base_corrections import TimeIntervalCorrection
+
+
+class BayesModel(TimeIntervalCorrection):
```

Comment:
Im not sure i like the name, its a bit undescriptive. A bayes model could refer to anything. Naive bayes is just the technique for generating the model right? so i feel the name should be more descriptive of what its actually modelling and then have a suffix of NaiveBayes or NB, no?